x-pack/legacy/plugins/ml/public/services/new_job_capabilities_service.ts

```diff
@@ -186,62 +178,22 @@ function mix(field: Field, agg: Aggregation) {
   field.aggs.push(agg);
 }
 
-function createCountFieldAndAggs() {
-  const countField: Field = {
+function processFieldlessAggs(aggs: Aggregation[], fields: Field[]) {
```

Comment:
you already have a function with this name in `general/utils` which deals with detectors 🤔